src/helpers/sueHelper.ts

```diff
@@ -1,24 +1,40 @@
-import * as appJson from '../../app.json';
-import { secrets } from '../config';
+import { storageHelper } from './storageHelper';
 
-const namespace = appJson.expo.slug as keyof typeof secrets;
-export const serverUrl = secrets[namespace]?.sue?.serverUrl;
-export const apiKey = secrets[namespace]?.sue?.apiKey;
-export const jurisdictionId = secrets[namespace]?.sue?.jurisdictionId;
+export const fetchSueEndpoints = async (serviceRequestId?: number) => {
+  const globalSettings = await storageHelper.globalSettings();
+  const { apiConfig = {} } = globalSettings?.settings?.sue || {};
 
-export const sueFetchObj = {
-  method: 'GET',
-  headers: {
-    accept: 'application/json',
-    api_key: apiKey
+  if (!apiConfig) {
+    return;
   }
-};
 
-export const suePrioritiesUrl = `${serverUrl}/priorities?jurisdiction_id=${jurisdictionId}`;
-export const sueRequestsUrl = `${serverUrl}/requests?jurisdiction_id=${jurisdictionId}`;
-export const sueRequestsUrlWithServiceId = (serviceRequestId: number) =>
-  `${serverUrl}/requests/${serviceRequestId}?jurisdiction_id=${jurisdictionId}`;
-export const sueServicesUrl = `${serverUrl}/services?jurisdiction_id=${jurisdictionId}`;
-export const sueStatusesUrl = `${serverUrl}/statuses?jurisdiction_id=${jurisdictionId}`;
+  const { apiKey, jurisdictionId, serverUrl } = apiConfig[apiConfig?.whichApi] || apiConfig;
+
+  const sueFetchObj = {
+    method: 'GET',
+    headers: {
+      accept: 'application/json',
+      api_key: apiKey
+    }
+  };
+
+  const suePrioritiesUrl = `${serverUrl}/priorities?jurisdiction_id=${jurisdictionId}`;
+  const sueRequestsUrl = `${serverUrl}/requests?jurisdiction_id=${jurisdictionId}`;
+  const sueRequestsUrlWithServiceId = `${serverUrl}/requests/${serviceRequestId}?jurisdiction_id=${jurisdictionId}`;
+  const sueServicesUrl = `${serverUrl}/services?jurisdiction_id=${jurisdictionId}`;
+  const sueStatusesUrl = `${serverUrl}/statuses?jurisdiction_id=${jurisdictionId}`;
 
-export const suePostRequest = `${serverUrl}/requests`;
+  const suePostRequest = `${serverUrl}/requests`;
```

Comment:
what about ordering them in the same way as in the return object?

Reply:
I wanted to sort alphabetically, but I didn't sort the variables. I will sort the variables alphabetically.